### PR TITLE
Angular remastered bootstrap

### PR DESCRIPTION
**Pull request by FrankIglesias**

### CODE REVIEW

File: .pug-lintrc.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+{
+  "extends": "pug-lint-config-wolox"
+}
```
Comment (SKOLZ) [inline .pug-lintrc.json]: empty line at eof

File: README.md [hunk 2]
```diff
@@ -71,10 +55,10 @@ To run these specs against your development environment, execute the following:
 ## Deploy
 
 #### S3
-In order to deploy you must first create **config/aws.js** file with the credentials of the Amazon S3 bucket for each environment.
+In order to deploy you must first create **aws.js** file with the credentials of the Amazon S3 bucket for each environment.
 The file needs to have to follow the format specified in *config/aws.js.example*
 
-Then just run `gulp s3 --env <environment name>` with your desired env as parameter.
```
Comment (SKOLZ) [inline README.md]: this command created the build and then deployed.

File: package.json [hunk 3]
```diff
@@ -2,12 +2,12 @@
   "name": "frontend-bootstrap",
   "version": "0.0.1",
   "engines": {
-    "node": "8.5.0",
+    "node": "8.npm au5.0",
```
Comment (SKOLZ) [inline package.json]: wut????? this is an error right?

Comment (SKOLZ) [reply package.json]: you were trying to run `npm audit` ? :P

File: src/app/app.config.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
+const angular = require('angular');
+
 angular.module('app-bootstrap').config([
   'localStorageServiceProvider',
-  function (localStorageServiceProvider) {
+  function () {
 
     // Local Storage Setup
-    localStorageServiceProvider.setPrefix(window.btoa('app-bootstrap-/* @echo environment */'));
+    // localStorageServiceProvider.setPrefix(window.btoa('app-bootstrap-/* @echo environment */'));
```
Comment (SKOLZ) [inline src/app/app.config.js]: why is this commented? do we need it?

File: .eslintrc [hunk 5]
```diff
@@ -1,177 +1,177 @@
 {
```
Comment (SKOLZ) [inline .eslintrc]: we should extend `eslint-config-wolox` if necessary we should create an angular config

**Comment by SKOLZ** [comment]
we should add stylelint too

**Comment by FrankIglesias** [comment]
This pr will move to angularjs-bootstrap

**Pull request closed without merging**